docs/cassandra.md

```diff
@@ -1,41 +1,13 @@
-# `cassandra` Metrics Receiver
+# Cassandra
 
-The `cassandra` metrics receiver can fetch stats from a Cassandra node's Java Virtual Machine (JVM) via [JMX](https://www.oracle.com/java/technologies/javase/javamanagement.html).
+Follow https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/cassandra
+for instructions to collect logs and metrics from this application using Ops Agent.
```

Comment:
Follow the [installation guide](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/cassandra) for instructions to collect logs and metrics from this application using Ops Agent.

Reply:
Sure thing. Changed.